src/gui/dialogs/DialogLoadUnload.cpp

```diff
@@ -0,0 +1,102 @@
+#include "DialogLoadUnload.hpp"
+#include "DialogLoadUnload.h"
+#include "gui.h"      //resource_font
+#include "resource.h" //IDR_FNT_BIG
+
+#ifdef __cplusplus
+extern "C" {
+#endif //__cplusplus
+int16_t WINDOW_CLS_DLG_LOADUNLOAD = 0;
```

Comment:
I would prefer to use ALL UPPERCASE only for preprocessor macros.

Reply:
all window class IDs has this format